course2_react/todo-list/src/List.js

```diff
@@ -0,0 +1,38 @@
+const List = ({ names, deleteName, setNames, input, setInput }) => {
+  //const handeledIt = deleteName();
+
+  const addName = () => {
+    let newId = names.length === 0 ? 1 : names[names.length - 1].id + 1;
```

Comment:
We can have some different logic as well for generating unique ids. Search for how we can generate unique numbers in javascript. You can create a function for it & just call wherever required